README.markdown

```diff
@@ -1153,6 +1153,7 @@ Directives
 * [lua_ssl_protocols](#lua_ssl_protocols)
 * [lua_ssl_trusted_certificate](#lua_ssl_trusted_certificate)
 * [lua_ssl_verify_depth](#lua_ssl_verify_depth)
+* [lua_ssl_conf_command](#lua_ssl_conf_command)
```

Comment:
Need to sync the change to https://github.com/openresty/lua-nginx-module/blob/master/doc/HttpLuaModule.wiki